curvefs/src/client/s3/disk_cache_base.cpp

```diff
@@ -88,7 +88,7 @@ std::string DiskCacheBase::GetCacheIoFullDir() {
 int DiskCacheBase::CreateDir(const std::string dir) {
     size_t p = dir.find_last_of('/');
     std::string dirPath = dir;
-    if (p != -1) {
+    if (p != -1ULL) {
```

Comment:
```suggestion
    if (p != std::string::npos) {
```

You can fix this in the next PR if you want.

And we should finish this PR quickly, otherwise, when someone else merges PRs, they may introduce new warnings. :anger: